config/crds/migration_v1alpha1_migcluster.yaml

```diff
@@ -58,8 +58,15 @@ spec:
               type: string
           required:
           - isHostCluster
+          - insecure
```

Comment:
Believe this needs to be removed

Reply:
Looks like the CRD was edited manually instead of updating the Field tag.  Fixed.